md/portal-user-case-list.md

```diff
@@ -0,0 +1,96 @@
+# User Case list in Bonita Portal
+
+This page explains what a user with the _User_ profile in Bonita Portal can see and do about cases (the process instances).
+
+_Users_ can view a list of the cases they are involved in, filter cases, and search for a case.  
+Then on a given case, they can view the case overview, access the case details, and access the list of tasks to do and/or done in the case.
+
+## Case list
+
+Here is a view of the User case list.
+![User Case list](images/UI2021.1/user-case-list.png)<!--{.img-responsive}-->
+
+This list displays the cases in which the logged user is involved: 
+  * They started the case, from the [process list](user-process-list.md) or an application
+  * AND/OR they have done tasks in this case earlier, from the [task list](user-task-list.md) or an application
+
+### Open and Archived cases
+
+At the top of the page, two tabs act as the first filter of the list: _Open cases_ and _Archived cases_.  
+Obvious differences between the two:
+  * Open cases have an _Available tasks_ column but not the archived cases
+  * Archived cases have an _End date_ but not the open cases
+  
+Less obvious difference:
+Users can add a comment in an open case, but cannot anymore in an archived case.  
+Comments are discribed a little below in this page.
+
+### Filter by process
+The default filtering option is "All", but you can click on the drop down list widget to view all process names and select the one of your interest.
+Once clicked, the list is refreshed and only displays the cases of the selected process.
+
+### Filter by initiator
+The default filtering option is "Anyone", but you can click on the drop down list widget to view the second option "Me".  
+If you select "Me", the list is refreshed and only displays the cases you have started.
+
+### Filter by search key
+
+::: info
```

Comment:
I am not sure we need the second sentence of this info note

Reply:
ok